app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -160,6 +167,9 @@ private void onDrawerItemClick(final MenuItem item) {
             PrefUtils.setShouldNotOpenDrawerOnStart(GdgNavDrawerActivity.this);
         }
 
+        Bundle b = new Bundle();
+        b.putString(Const.DRAWER_ITEM, item.getTitle().toString());
```

Comment:
I did the same thing in the latest develop branch. Can you rebase with the latest develop branch and update these lines. 
I like the `instanceof` check.